### ISSUE TITLE
Remake CNN from Keras to Tensorflow

### ISSUE THREAD

**Issue by mike2ox** [KIST-Iceberg/Iceberg#3]
Keras로 만든 모델을 Tensorflow 버전으로 remake

**Comment by mike2ox** [comment]
1.  layer 1에서 zero padding -> vaild padding 사용 : output height, width size를 clear하게 도출
2. filter size / strides / batch size

- first : [4, 4] / [2, 2] / 64, reminder : [3, 3] / [2, 2] / (128 -> 256) , last : [3, 3] / [3, 3] / 512
3. X, Y placeholder -> truncated_nomal 사용

**Comment by mike2ox** [comment]
CNN에서 동일 물체가 회전했을 때의 데이터도 고려?
> https://www.kaggle.com/henokanh/cnn-batchnormalization-0-1646
> https://www.nextobe.com/single-post/2017/11/02/CapsNetCapsule-Network (dataset이 적을 경우 CNN의 문제점)

**Comment by mike2ox** [comment]
standard NN과 다른 NN
> 1. capsule net
https://hackernoon.com/capsule-networks-are-shaking-up-ai-heres-how-to-use-them-c233a0971952
https://www.nextobe.com/single-post/2017/11/02/CapsNetCapsule-Network
https://newtypeunion.com/2017/11/27/capsule-networks/
> 2.  inception modul
https://norman3.github.io/papers/docs/google_inception.html

**Comment by kairos03** [comment]
형 저 파리에서 폰 잃러버료서 gurigyuri로 카톡점주세여

**Comment by mike2ox** [comment]
the end
